lib/tennis_game.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+# TennisGame: used to manage the scoring in a tennis game
+class TennisGame
+  TENNIS_POINTS = %w(Love Fifteen Thirty Fourty).freeze
+  attr_accessor :points
+  def initialize
+    # create score array with scores for both teams
+    @points = [0, 0]
+  end
+
+  # converts the score into a the tennis scores
+  def score
+    if deuce?
+      'Deuce'
+    # if both teams have at least 3 points but are not equal
+    elsif advantage?
```

Comment:
If you add `Advantage` to the list of `TENNIS_POINTS` then you probably don't need the `advantage` logic - you could just show in the `else` logic, e.g. `40 : Advantage`. Other logic would have to be amended so we don't have a scenario where two players can have `Advantage` but it makes logical sense.